### PR TITLE
Add RawTherapee (a raw image processing program)

### PR DESCRIPTION
**Pull request by tokiedokie**
## RawTherapee
create `rawtherapee.json`
RawTherapee is  a raw image processing program
ver 5.7

### CODE REVIEW

File: bucket/rawtherapee.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "homepage": "https://rawtherapee.com/",
```
Comment (Ash258) [inline bucket/rawtherapee.json]:
```suggestion
    "homepage": "https://rawtherapee.com",
```

File: bucket/rawtherapee.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "description": "RawTherapee is a free, cross-platform raw image processing program",
+  "license": "GPL v3",
```
Comment (Ash258) [inline bucket/rawtherapee.json]:
```suggestion
    "license": "GPL-3.0-only",
```

File: bucket/rawtherapee.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "homepage": "https://rawtherapee.com/",
+  "description": "RawTherapee is a free, cross-platform raw image processing program",
```
Comment (Ash258) [inline bucket/rawtherapee.json]:
```suggestion
  "description": "Raw image processor.",
```

File: bucket/rawtherapee.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  "license": "GPL v3",
+  "version": "5.7",
+  "url": "https://rawtherapee.com/shared/builds/windows/RawTherapee_5.7_WinVista_64.zip",
+  "hash": "f6d78951a5a398e259637ff8c3dadd5209bf6bde6694a3a3e168ba8bfd576316",
+  "installer": {
+    "script": "Expand-InnoArchive -Path \"$original_dir\\RawTherapee_5.7_WinVista_64.exe\" -DestinationPath \"$original_dir\""
```
Comment (Ash258) [inline bucket/rawtherapee.json]:
```suggestion
        "script": "Get-ChildItem \"$dir\" 'RawTherapee*WinVista*.exe' | Expand-InnoArchive -DetinationPath \"$dir\""
```

File: bucket/rawtherapee.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  "bin": [
```
Comment (Ash258) [inline bucket/rawtherapee.json]:
```suggestion
    "bin": [
        "rawtherapee-cli.exe",
         [
            "rawtherapee-cli.exe",
            "rawtherapee"
        ]
    ],
```

File: bucket/rawtherapee.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    ],
+    [
+      "rawtherapee-cli.exe",
+      "rawtherapee-cli"
+    ]
+  ],
+  "shortcuts": [
+    [
+      "rawtherapee.exe",
+      "RawTherapee 5.7"
```
Comment (Ash258) [inline bucket/rawtherapee.json]:
```suggestion
      "RawTherapee"
```

File: bucket/rawtherapee.json (hunk 5 above)
Comment (tokiedokie) [reply bucket/rawtherapee.json]: according to https://rawpedia.rawtherapee.com/Command-Line_Options
rawtherapee and rawtherapee-cli are different

File: bucket/rawtherapee.json (hunk 4 above)
Comment (tokiedokie) [reply bucket/rawtherapee.json]: updated
see ad6c17e

File: bucket/rawtherapee.json (hunk 5 above)
Comment (Ash258) [reply bucket/rawtherapee.json]: Both have argument handler?

If yes, then use just `"bin": "["rawtherapee-cli.exe", "rawtherapee.exe"]` No need to rename shim with same name as original.

Comment (tokiedokie) [reply bucket/rawtherapee.json]: yes both have it

updated
see 2a8fee2

Comment (Ash258) [reply bucket/rawtherapee.json]: > Add hash extraction from https://rawtherapee.com/2019/09/repackaged-windows-build/

Comment (tokiedokie) [reply bucket/rawtherapee.json]: I found sha256sum in https://rawtherapee.com/shared/builds/windows/

so i added autoupdate.hash

see dae95c8

**Pull request merged**